lib/private/BackgroundJob/JobList.php

```diff
@@ -214,7 +214,7 @@ public function getJobsIterator($job, ?int $limit, int $offset): iterable {
 	 * Get the next job in the list
```

Comment:
```suggestion
	 * Get the next job in the list
	 * @param class-string<IJob>[] $jobClasses
```
Or something like that

Reply:
Actually should be in IJobList instead and `@inherit` here.